### PR TITLE
Make current saved address module accessible from both Wallet section and Wallet Settings section

### PR DESCRIPTION
**Pull request by saledjenic**
This PR introduces the necessary changes for reusing saved addresses on the Wallet Settings side.

Among that this PR also fixes:
- inconsistent position of the "Add new address" button on the Wallet section side
- inappropriate content on the right side for the selection on the left side of the Wallet section when the user is switching main sections and then lands on the Wallet section
- the position of the delete saved address popup was based on the displayed saved address, which means, that if you're deleting a saved address placed at the bottom of the list, the popup will be displayed in the bottom right part of the app, instead to be centered as it's now
- reduce redundancy

Closes: #13088

### CODE REVIEW

File: ui/app/AppLayouts/Profile/views/WalletView.qml [hunk 1]
```diff
@@ -26,8 +27,8 @@ SettingsContentBase {
     id: root
 
     property var emojiPopup
-    property var rootStore
-    property var walletStore
+    property ProfileSectionStore rootStore
```
[the remark this answers is not in the document]
Comment (saledjenic) [reply ui/app/AppLayouts/Profile/views/WalletView.qml:30]: Once we come to Storybook with it we can change it to `var` or find a better solution, creating a fake `ProfileSectionStore` type or so, but having a strict type is always better than having `var` everywhere.

File: ui/app/AppLayouts/Wallet/views/RightTabBaseView.qml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    property var sendModal
+
+    property alias header: header
+    property alias headerButton: header.headerButton
+    property alias networkFilter: header.networkFilter
+
+    default property Item content
+
+    Component.onCompleted: {
+        content.parent = contentWrapper
```
[the remark this answers is not in the document]
Comment (saledjenic) [reply ui/app/AppLayouts/Wallet/views/RightTabBaseView.qml:25]: Pretty sure that's possible the way you've suggested. `content` is the `default` property.

File: ui/app/mainui/AppMain.qml [hunk 3]
```diff
@@ -1668,6 +1679,132 @@ Item {
         }
     }
 
+    Loader {
```
[the remark this answers is not in the document]
Comment (saledjenic) [reply ui/app/mainui/AppMain.qml:1682]: I agree, but that should and will be a part of AppMain.qml refactor, which is going to happen very soon.

File: ui/app/AppLayouts/Wallet/views/RightTabBaseView.qml (hunk 2 above)
[the remark this answers is not in the document]
Comment (saledjenic) [reply ui/app/AppLayouts/Wallet/views/RightTabBaseView.qml:25]: Yes, that can work.

[the remark this answers is not in the document]
Comment (saledjenic) [reply ui/app/AppLayouts/Wallet/views/RightTabBaseView.qml:25]: @Cuteivist so what about this?

[the remark this answers is not in the document]
Comment (saledjenic) [reply ui/app/AppLayouts/Wallet/views/RightTabBaseView.qml:25]: I see it as two approaches for the same thing. 
In both the content is the default property.

> I think default property alias would work here much better.

Please let me know what are advantages of one over another? 

And let me know if this is a change request?

**Review by alaibe — APPROVED** [review]

**Review by dlipicar — APPROVED** [review]

**Pull request merged**